tests/test_carbon_handler.cairo

```diff
@@ -98,6 +99,27 @@ fn setup_project(
     project.set_project_carbon(project_carbon);
 }
 
+/// Mint shares without the minter contract. Testing purposes only.
```

Comment:
It's better to deploy the minter here because this test will fail once we add access control or disable batch_mint.

Reply:
Will be done in next test PR